aquality-tracking-integrations-core/src/main/java/aquality/tracking/integrations/core/IHttpClient.java

```diff
@@ -0,0 +1,13 @@
+package aquality.tracking.integrations.core;
+
+import org.apache.http.Header;
+
+import java.net.URI;
+import java.util.List;
+
+public interface IHttpClient {
```

Comment:
are you sure we need it?

Reply:
As I use DI, yes, it'd be better to have interface.